ha_mroonga.cpp

```diff
@@ -12495,12 +12496,14 @@ int ha_mroonga::generic_store_bulk_new_date(Field* field, grn_obj* buf)
   MRN_DBUG_ENTER_METHOD();
   int error = 0;
   bool truncated = false;
-  Field_newdate* newdate_field = (Field_newdate*)field;
-  MYSQL_TIME mysql_date;
-  MRN_FIELD_GET_TIME(newdate_field, &mysql_date, current_thd);
-  mrn::TimeConverter time_converter;
-  long long int time =
-    time_converter.mysql_time_to_grn_time(&mysql_date, &truncated);
+  long long int time = 0;
+  if (!field->is_null()) {
+    Field_newdate* newdate_field = (Field_newdate*)field;
```

Comment:
```suggestion
    auto newdate_field = static_cast<Field_newdate*>(field);
```

せっかくなので新しい書き方にしましょうか。

Reply:
fix: [3e917e5](https://github.com/mroonga/mroonga/pull/807/commits/3e917e528aba226d8f1cc55f07242ea6a679329f) はい！